etw/README.md

```diff
@@ -0,0 +1,52 @@
+# Etw Provider for Java Codeless Agent
+
+Provider Name: **`Microsoft-ApplicationInsights-Java-IPA`**
+Provider GUID: **`1f0dc33f-30ae-5ff3-8b01-8ca9b8509233`**
+
+# Building
+
+## Build Requirements
+To build the native resources, you'll need
+* Visual Studio Build Tools, C++
+* Windows 10 SDK (developed with `10.0.18362.0`)

```

Comment:
nit: it may be possible to include a script that installs the required deps using chocolatey. Chocolatey is available in GitHub actions, as well as it's easy to install it with a PowerShell script.
Sharing some thoughts here:

Install choco if it's not installed:
```
@"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "[System.Net.ServicePointManager]::SecurityProtocol = 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
```

Install Tools and SDK using choco:
```
choco install -y --includeOptional visualstudio2017-workload-netcoretools
choco install -y windows-sdk-10-version-1809-all
```

Note: this installation step takes about 6-10 minutes on clean machine.

Reply:
This is a good idea. I might do this in the future.